packages/backend/src/tests/tigerbeetle.ts

```diff
@@ -10,9 +10,10 @@ export async function startTigerbeetleContainer(
   clusterId: number = Config.tigerbeetleClusterId
 ): Promise<StartedTestContainer> {
   const { name: tigerbeetleDir } = tmp.dirSync({ unsafeCleanup: true })
+  // TODO const tigerBeetleFile = `${TIGERBEETLE_DIR}/cluster_${clusterId}_replica_0_test.tigerbeetle`
```

Comment:
```suggestion
```

Reply:
Refer to: https://github.com/interledger/rafiki/issues/518